lib/parallel_tests/test/runner.rb

```diff
@@ -95,8 +95,9 @@ def execute_command_and_capture_output(env, cmd, silence)
 
           output = open("|#{cmd}", "r") { |output| capture_output(output, silence) }
           exitstatus = $?.exitstatus
+          seed = output[/seed (\d+)/,1]
```

Comment:
https://github.com/seattlerb/minitest seems to print out this: 

```
% ruby -Ilib:test test/minitest/test_minitest_unit.rb
Run options: --seed 37685

# Running:
```

So probably better to leave as is?